# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openssl/sha_size_t.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- openssl-0.9.8/crypto/sha/sha.h.orig	2005-07-14 21:00:11.000000000 +0200
+++ openssl-0.9.8/crypto/sha/sha.h	2005-07-14 21:00:28.000000000 +0200
@@ -59,6 +59,7 @@
 #ifndef HEADER_SHA_H
 #define HEADER_SHA_H
 
+#include <stddef.h>
 #include <openssl/e_os2.h>
 
 #ifdef  __cplusplus
